### DISCUSSION TITLE
Artisan v2.8.4 released

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#1205] [Announcement]
[A v2.8 maintenance release](https://artisan-roasterscope.blogspot.com/2023/06/artisan-v284.html).

**Comment by Terracotta-62** [comment]
How to operate Hide and Show ？
toggle hide-background per click on background profile name (subtitle) during roasting. 
thank you !

**Reply by MAKOMO** [reply]
Just double-click on the background profiles name in the upper right corner. I updated the change history accordingly.

**Comment by Kikeooo** [comment]
Hello, this is my first post, I am a user of an fluid bed  and I want to know if my equipment is compatible with artisan software, the equipment has a full gauge controls interface installed.
Gracias.

**Reply by poundy** [reply]
Artisan can do lots of things - but guessing details of your roaster isn't one. 
Artisan supported machines (usually through working with the roaster vendor to fully support these machines) can be found at https://artisan-scope.org/machines/
Artisan supported devices (usually through a contributrion from the community who have such a device and a need to leverage it) can be found at https://artisan-scope.org/devices/index
